public/static/config.js

```diff
@@ -1,5 +1,5 @@
 var APP_CONFIG = {
```

Comment:
It's better to create sample config than add placeholders. You can create sample table for that and its id to config